src/lesson-1-8-demo/Multiplier.jsx

```diff
@@ -54,14 +40,15 @@ const DoubleCounter = () => {
     </div>
   );
 };
+export default Multiply;
 
 const Button = memo(({ onClick, label }) => {
```

Comment:
Notice that `Button` is memoized using `memo`. This means the `Button` will only re-render if the props, `onClick` or `label`, change